README-de.md

```diff
@@ -37,6 +37,12 @@ Um die Kubernetes-Website lokal laufen zu lassen, empfiehlt es sich, ein speziel
 
 > Wenn Sie die Website lieber lokal ohne Docker ausführen möchten, finden Sie weitere Informationen unter [Website lokal mit Hugo ausführen](#Die-Site-lokal-mit-Hugo-ausführen).
 
+Das benötigte Docsy Hugo Theme muss als git submodule installiert werden:
+
+```
+git submodule update --init --recursive --depth 1
```

Comment:
```
#Füge das Docsy submodule hinzu
git submodule update --init --recursive --depth 1
```

Reply:
the whole thing with the comment before as one code block :)

Reply:
Done